components/worker/.env.example

```
@@ -2,16 +2,26 @@
 VERSION=v1.0.0
+LOG_LEVEL=debug
 
-# HEALTH SERVER (liveness + readiness probes)
+# Health Server — liveness and readiness probes
 HEALTH_PORT=4006
 
-# RABBITMQ
+# MongoDB
+#MONGO_URI=mongo+srv
+MONGO_URI=mongodb
+MONGO_HOST=reporter-mongodb
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_HOST key should go before the MONGO_URI key

(UnorderedKey)